addons/ai/fnc_taskDefend.sqf

```diff
@@ -32,6 +32,7 @@ _group = [_group] call CBA_fnc_getGroup;
 if !(local _group) exitWith {}; // Don't create waypoints on each machine
 
 _position = [_position,_group] select (_position isEqualTo []);
+_position = _position call CBA_fnc_getPos;

```

Comment:
What about having that in the second element of the array above. Like so:
`_position = [_position, _group call CBA_fnc_getPos] select (_position isEqualTo []);